azure-pipelines.release.yml

```diff
@@ -142,6 +156,12 @@ extends:
 
           - name: UseServiceBusFake
             value: false
+          
+          # Feature Flags
```

Comment:
It probably worth getting the consistence and only use featureToggle or featureFlag as the discrepancy is slight widespread now including in the pipeline.